app/pages/HasNirmsUpdatePage.scala

```diff
@@ -16,25 +16,12 @@
 
 package pages
 
-import models.UserAnswers
 import play.api.libs.json.JsPath
 
-import scala.util.Try
-
 case object HasNirmsUpdatePage extends QuestionPage[Boolean] {
 
   override def path: JsPath = JsPath \ toString
 
   override def toString: String = "hasNirmsUpdate"
 
-  override def cleanup(
```

Comment:
Cleanup not needed at this point anymore. Cleanup happens after submission on CYA. Having this cleaned up here causes the NirmsNumber to be prematurely removed on the nirms number journey

Reply:
does removing this affect how the create profile journey works?

Reply:
Shouldn't do as its the update page? (I had the same thought when reviewing yesterday! :) )

Reply:
As far as I can tell, no.